### PR TITLE
Ria 3799

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-3799

### Change description ###

Ria 3799

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by tarun-palisetty — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@ featureFlag:
   isEmStitchingEnabled: ${IA_EM_STITCHING_ENABLED:false}
   isSubmitHearingRequirementsEnabled: ${IA_SUBMIT_HEARING_REQUIREMENTS_ENABLED:true}
   timedEventServiceEnabled: ${IA_TIMED_EVENT_SERVICE_ENABLED:false}
-  isfeePaymentEnabled: ${IA_FEEPAYMENT_ENABLED:false}
+  isfeePaymentEnabled: ${IA_FEEPAYMENT_ENABLED:true}
```
Comment (divisathyan) [inline src/main/resources/application.yaml:92]: You may want to keep this false, as it will merge to master

**Review by divisathyan — CHANGES_REQUESTED** [review]
Please check comment

**Review by Lee365 — APPROVED** [review]
LGTM

**Comment by mkayad** [comment]
jenkins failing with an exception

**Comment by mkayad** [comment]
we looked at the reason why the PR is not green and Marcin concluded the test will be failing until we release notification change to AAT and said that  this is acceptable exception because of notifications-api dependency

**Review by tarun-palisetty — DISMISSED** [review]

**Review by jkumar-moj — DISMISSED** [review]

**Review by divisathyan — APPROVED** [review]

**Review by jkumar-moj — APPROVED** [review]

**Pull request merged**